core/src/test/java/org/pentaho/metaverse/analyzer/kettle/step/excelinput/ExcelInputStepAnalyzerTest.java

```diff
@@ -199,4 +199,9 @@ public void resourcesFromRowGotSuccessfullyWhenStepInputMetaInterfaceIsNull() th
     assertEquals( 1, resources.size() );
   }
 
+  @Test
+  public void testNewInstance(){
```

Comment:
It would be good to mimic the changes made to the SelectValuesStepAnalyzerTest here, by extending ClonableStepAnalyzerTest and implementing the newInstance method - this will ensure that the cloneAnalyzer method is tested properly. This applies to all the tests in this PR. Here's a link to the change for reference: 
https://github.com/pentaho/pentaho-metaverse/pull/524/files#diff-a2679a1a5482329241673143be6af01cR53